sydent/http/auth.py

```diff
@@ -51,7 +59,11 @@ def tokenFromRequest(request):
     return token
 
 
-def authV2(sydent, request, requireTermsAgreed=True):
+def authV2(
+    sydent: "Sydent",
+    request: 'Request',
+    requireTermsAgreed: bool = True,
+) -> Optional['Account']:
```

Comment:
I changed it to 'Account`: should I change the docstring as weel? It looks like according to docstring it can return `none` for v1 accounts, but we are dropping support for v1 accounts?

Reply:
Sure, go ahead and update the docstring. It looks like the `None` path was removed in ce04a68950689fd9866c88748c70f5a6796bd1e8, so I think we're fine removing mention of it.